packaging/fpm/README.md

```diff
@@ -0,0 +1,45 @@
+# FPM
+
+[FPM](https://github.com/jordansissel/fpm) is a tool to easily and quickly build packages such as rpms, debs, etc.
+
+## Requirements
+
+To build this container, you will need the following:
+* Singularity installed
+* Internet access
+* Root access
+
+## Build instructions
+
+To build the container, use:
+
+`sudo singularity build fpm.simg fpm.def`
```

Comment:
can you change `fpm.simg` to `fpm.sif` throughout to reflect the new SIF format in 3.0?